### PR TITLE
log scaling and logging

### PR DESCRIPTION
**Pull request by cynicalsavant**
just added some log scaling adn additional logging before running the new experiment

### CODE REVIEW

File: projects/super_res/config.py [hunk 1]
```diff
@@ -21,15 +21,16 @@
 tensorboard_dir = "./tensorboard"
 milestone = 1
 
-batch_size = 2
+batch_size = 4
```
Comment (gideonite) [inline projects/super_res/config.py:24]: nit: Please revert this change. I don't think this needs to be committed.

File: projects/super_res/data/vsrdata.py [hunk 2]
```diff
@@ -22,6 +22,10 @@ def __init__(self, channels, mode, length):
         c384_np = np.stack([c384[channel].values for channel in channels], axis = 2)
         c48_np = np.stack([c48[channel].values for channel in channels], axis = 2)
 
+        if logscale:
+            c384_np = np.log(c384_np - c384_np.min() + 1e-14)
```
Comment (gideonite) [inline projects/super_res/data/vsrdata.py:26]: Do you know what floating point precision we are using? It's probably better to make this more robust via finfo, e.g. if we are using `torch.float32`

```
 torch.finfo(torch.float32).resolution
```
This returns 1e-06.

File: projects/super_res/model/autoreg_diffusion.py [hunk 3]
```diff
@@ -1312,6 +1311,7 @@ def sample(self):
         self.ema.ema_model.eval()
 
         cmap = mpl.colormaps['viridis']
+        sm = smap(None, cmap)
```
Comment (gideonite) [inline projects/super_res/model/autoreg_diffusion.py:1314]: I'm a little confused here. What if `config.data_config.logscale = False`? Does this mean that we don't want to plot in logscale anymore?

File: projects/super_res/model/autoreg_diffusion.py [hunk 4]
```diff
@@ -1329,9 +1329,6 @@ def sample(self):
                     #videos, base, nsteps, flows = self.ema.ema_model.sample(lres, hres, True)
                     videos, base, nsteps, flows = self.ema.ema_model.sample(lres, hres)
 
-                    vmax = torch.max(torch.max(videos.max(), hres.max()), lres.max()).cpu().numpy()
```
Comment (gideonite) [inline projects/super_res/model/autoreg_diffusion.py:1332]: nit: I know you are removing this code, but I think you can use python's native `max` function here, e.g. 
```
max([videos.max(), hres.max(), lres.max()])
```
which I think would help clarify this line.

**Review by gideonite — CHANGES_REQUESTED** [review]
This is a great and simple idea which seems to have made a big difference. 

A few "nits" which are just small comments that I'm making which I think would help improve this PR, but which you can ignore if you choose. 

I am little confused about the logging and setting the jiggle parameter (`eps`) to match the floating point precision that we are using in the code. Once these two things are addressed, I think we'll be good to go.

File: projects/super_res/model/autoreg_diffusion.py (hunk 3 above)
Comment (cynicalsavant) [reply projects/super_res/model/autoreg_diffusion.py:1314]: that's just the default value in the function. it takes the value from config file where it's set to be true

File: projects/super_res/model/autoreg_diffusion.py (hunk 4 above)
Comment (cynicalsavant) [reply projects/super_res/model/autoreg_diffusion.py:1332]: aah okay

File: projects/super_res/data/vsrdata.py (hunk 2 above)
Comment (gideonite) [reply projects/super_res/data/vsrdata.py:26]: Ok got it. Then you can do `np.finfo(arr.dtype).resolution`. What do you think?

File: projects/super_res/model/autoreg_diffusion.py (hunk 3 above)
Comment (gideonite) [reply projects/super_res/model/autoreg_diffusion.py:1314]: Sorry, I still don't get it. What happens if the `config.logscale` is False? I think the data will not be plotted in logscale but we always want to plot logscale, right?

Comment (cynicalsavant) [reply projects/super_res/model/autoreg_diffusion.py:1314]: not just for plotting but as an input to the model itself

**Comment by gideonite** [comment]
We are going to forgo having plots with backward compatibility, assuming that moving forward we are going to log scaling the relevant features. 

I'm going to approve now just to unblock you, but would appreciate the change regarding the jiggle parameter `eps` to the log transform of the data.

**Comment by gideonite** [comment]
There were some conflicts with my PR for changing the config. In sympathy, I resolved them since I know that this can be super frustrating. Please double check that I did a proper job. @cynicalsavant

**Review by gideonite — APPROVED** [review]
